tests/trainer/test_checkpoint.py

```diff
@@ -29,7 +31,10 @@
 from composer.trainer import trainer
 from composer.trainer.trainer import Trainer
 from composer.utils import dist, is_tar, reproducibility
-from composer.utils.checkpoint import _ensure_valid_checkpoint, glob_filter
+from composer.utils.checkpoint import _ensure_valid_checkpoint  # type: ignore
+from composer.utils.checkpoint import _write_checkpoint_file  # type: ignore
+from composer.utils.checkpoint import _COMPOSER_STATES_FILENAME, glob_filter
```

Comment:
```suggestion
from composer.utils.checkpoint import _ensure_valid_checkpoint, _COMPOSER_STATES_FILENAME, glob_filter, _write_checkpoint_file
```
nit: combine into one import

why is the type ignore needed? If it's unused, can we skip import?

Reply:
more typechecking issues that may just be a problem with my setup.